tests/php/Forms/HTMLEditor/HTMLEditorSanitiserTest.php

```diff
@@ -13,6 +13,12 @@ class HTMLEditorSanitiserTest extends FunctionalTest
 
     public function testSanitisation()
     {
+        // ensure configurable value matches what's used in this test
+        $config = HTMLEditorConfig::get('htmleditorsanitisertest');
+        $sanitiser = Injector::inst()->get(HTMLEditorSanitiser::class, false, [$config]);
+        $defaultValue = $sanitiser->config('link_noopener_value');
+        $sanitiser->config('link_noopener_value')->set('link_noopener_value', 'noopener, noreferrer');
```

Comment:
This can be `Config::inst()->set(HTMLEditorSanitiser::class, 'link_noopener_value', 'noopener, noreferrer');`
Due to the test being for that concrete class, and config being nested by `SapphireTest`.

Reply:
updated